### ISSUE TITLE
relevant papers have been published

### ISSUE THREAD

**Issue by Lkkcharck** [lipeiyu123/GuidanceLaw#1]

**Comment by Lkkcharck** [comment]
Hello, may I ask if your research background is active defense and relevant papers have been published

**Comment by lipeiyu123** [comment]
Yes, the background of this project is about active defense for aircraft. Currently, no papers have been published for this project because the effect that the current model can achieve is only close to the traditional PD guidance law. My teacher told me:  PID is the best way and your algorithm will not better than PID.

**Comment by Lkkcharck** [comment]
Do you have any primary code that can be run, I want to refer to writing, I wrote a proportional guide code in your framework, the running result is not good

**Comment by ydengfeng** [comment]
This project still cannot run after adding the dependency environment. When running the train script, pycharm prompts that the parl module is missing Model. What is going on? What modifications need to be made before this project can run?

**Comment by ydengfeng** [comment]
> Do you have any primary code that can be run, I want to refer to writing, I wrote a proportional guide code in your framework, the running result is not good

Can you share a project that can be run? I would like to learn from it.
